AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1080,7 +1080,14 @@ - (void)__layout
     // measureWithSizeRange: on subnodes to assert.
     return;
   }
+
+  // Handle placeholder layer creation in case the size of the node changed after the initial placeholder layer
```

Comment:
@maicki could you explain just a few words more on why this is necessary? I looked around and didn't see an obvious place at which we destroy the placeholder layer, so I assume the main requirement is to add it to the view hierarchy again, or to update the contents of the layer.

However, updating the contents will be skipped if it already has a contents. Is the contents proactively destroyed when the size of the node changes?